HUD/IsPausemapInInteriorMode.md

```diff
@@ -0,0 +1,13 @@
+---
+ns: HUD
+aliases: ["0x9049FE339D5F6F6F", "_IS_MINIMAP_IN_INTERIOR"]
+---
+## IS_PAUSEMAP_IN_INTERIOR_MODE
+
+```c
+// 0x9049FE339D5F6F6F 0x199DED14
+BOOL IS_PAUSEMAP_IN_INTERIOR_MODE();
+```
+
+## Return value
+Returns if the map is in an interior.
```

Comment:
No need to have a return value description when native name is obvious.

Reply:
okay, thank you!